cluster/azure/aks-gitops/outputs.tf

```diff
@@ -1,3 +1,7 @@
 output "kubeconfig_done" {
   value = "${module.aks.kubeconfig_done}"
 }
+
+output "cluster_derived_resource_group" {
+  value = "MC_${var.resource_group_name}_${var.cluster_name}_${var.resource_group_location}"
```

Comment:
Instead of concatenating, I would suggest using `node_resource_group` exported attribute from `azurerm_kubernetes_cluster` terraform provider.